ruby-saml.gemspec

```diff
@@ -25,28 +25,33 @@ Gem::Specification.new do |s|
   s.summary = %q{SAML Ruby Tookit}
   s.test_files = `git ls-files test/*`.split("\n")
 
-  s.add_runtime_dependency('uuid', ['~> 2.3'])
-  s.add_runtime_dependency('nokogiri', ['~> 1.5.0'])
+  s.add_runtime_dependency('uuid', '~> 2.3')
+  if RUBY_VERSION < '1.9'
+    # 1.8.7
+    s.add_runtime_dependency('nokogiri', '~> 1.5.10')
+  else
+    s.add_runtime_dependency('nokogiri', '~> 1.6')
```

Comment:
Shouldn't this be `~> 1.6.0`? Without using the _patch-level_ identifier, this runs the risk of v1.7+ breaking something in the future. Otherwise you could remove the conditional and just use `~> 1.5` and that should satisfy everything Ruby 1.8+.